lib/matplotlib/axes/_axes.py

```diff
@@ -5338,72 +5349,54 @@ def pcolor(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
         Other Parameters
         ----------------
         antialiaseds : bool, optional, default: False
-            The default ``antialiaseds`` is False if the default
-            ``edgecolors="none"`` is used.  This eliminates artificial lines
+            The default *antialiaseds* is False if the default
+            *edgecolors*\ ="none" is used.  This eliminates artificial lines
```

Comment:
I can't run the tests wth this line:  `SyntaxError: invalid escape sequence \`

Reply:
Strange. This is the [recommended way](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup):

     Use a backslash escaped space to work around that: thisis\ *one*\ word.

The docs itself look fine:
https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.pcolor.html?highlight=pcolor#matplotlib.axes.Axes.pcolor

Why is the test inspecting the docstring at all? Do we have some doctests somewhere that need to be run?

Reply:
@timhoffm The recommended way you cite is for Sphinx, while @jklymak's error is coming from Python. The backslash escape for Sphinx is on the text that it sees, which is either a reST file or a docstring (extracted to a reST file by auto-something), but in order to place the backslash in the docstring, you _also_ need to escape it for Python. So you need to double-escape or make it a raw string.

As to why @jklymak only sees it on one machine, maybe it's `tests.py` vs `pytest`? The former turns this deprecation warnings into errors. Also, the warnings are only raised on Python 3.6+.

Reply:
Ahh. Thanks @QuLogic :  `python tests.py` does not work and gives the Syntax Error.  `py.test` works.  Thats what I get for just hitting `^R` for the last time I ran the tests - obviously on one machine I mentally upgraded to `py.test` and the other I haven't :wink:...